frontend/src/components/CustomTimePicker/CustomTimePickerPopoverContent.tsx

```diff
@@ -37,12 +48,23 @@ function CustomTimePickerPopoverContent({
 	onSelectHandler,
 	handleGoLive,
 	selectedTime,
+	activeView,
+	setActiveView,
 }: CustomTimePickerPopoverContentProps): JSX.Element {
 	const { pathname } = useLocation();
 
 	const isLogsExplorerPage = useMemo(() => pathname === ROUTES.LOGS_EXPLORER, [
 		pathname,
 	]);
+	const urlQuery = useUrlQuery();
+	const activeTimezoneOffset = useMemo(() => {
+		const timezone = urlQuery.get('timezone') ?? TIMEZONE_DATA[0].value;
```

Comment:
Add safety check here

Reply:
This is old code, it has been changed it to https://github.com/SigNoz/signoz/blob/147e5d6166188488656a9c43cd91c3b9d59a0bf7/frontend/src/components/CustomTimePicker/CustomTimePickerPopoverContent.tsx#L55-L56